src/main/java/domain/user/Answer.java

```diff
@@ -0,0 +1,30 @@
+package domain.user;
+
+public class Answer {
+    private static final String YES = "y";
+    private static final String NO = "n";
+
+    private String answer;
+
+    public Answer(String answer) {
+        validateAnswer(answer);
+        this.answer = answer;
+    }
+
+    private void validateAnswer(String answer) {
+        boolean isYes = YES.equals(answer);
+        boolean isNo = NO.equals(answer);
+        if (!isYes && !isNo) {
+            throw new IllegalArgumentException("대답으로는 'y' 또는 'n' 만 입력할 수 있습니다.");
+        }
+    }
+
+    public boolean isYes() {
+        return answer.equals(YES);
```

Comment:
만약 answer가 ``null``이면 어떻게 될까요 ? 🤔 
아마도 null이 equals() 함수를 호출하게 되어 **NullPointerException** 이 발생할거에요 😅
```suggestion
        return YES.equals(answer);
```

Reply:
놓쳤던 부분인 것 같습니다. 꼼꼼한 피드백 감사합니다 :) 👍